src/js/redux/reducers/award/awardReducer.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * awardReducer.js
+ * Created by Emily Gullo 01/23/2017
+ **/
+
+const initialState = [];
```

Comment:
Why is this an array? It seems that you are only holding on to one award at a time, and that you have a child key to access that. (Also why do you need the child key if you're only holding the one award?)